vimrc

```diff
@@ -173,7 +173,9 @@ set spellfile=$HOME/.vim-spell-en.utf-8.add
 set complete+=kspell
 
 " Always use vertical diffs
-set diffopt+=vertical
+if &diff
+  set diffopt+=vertical
+endif
```

Comment:
That doesn't work but [this does](https://github.com/thoughtbot/dotfiles/issues/655#issuecomment-605019271)!